erts/doc/src/erlang.xml

```diff
@@ -6154,6 +6154,66 @@ receive_replies(ReqId, N, Acc) ->
 
     <func>
       <name name="process_flag" arity="2" clause_i="1"
+	    anchor="process_flag_async_dist" since="OTP @OTP-18374@"/>
+      <fsummary>
+        Enable or disable truly asynchronous distributed signaling
+        for the calling process.
+      </fsummary>
+      <desc>
+        <p>
+          Enable or disable <i>truly asynchronous distributed signaling</i>
+          for the calling process. When disabled, the process sending a
+          distributed signal will block in the send operation if the buffer
+          for the distribution channel reach the
+          <seecom marker="erts:erl#+zdbbl">distribution buffer busy
+          limit</seecom>. The process will remain blocked until the buffer
+          shrinks enough. This might in some cases take a substantial amount
+          of time. When <i>truly asynchronous distributed signaling</i> is
+          enabled, operations sending asynchronous distributed signals will
```

Comment:
No need to repeat "truly asynchronous distributed signaling" that many times. Is enough to mention it once and then refer to the name of the flag `async_dist` Or maybe call the flag `async_dist_signaling`?

How much is buffered when async_dist = true? Is there a system_limit? configurable? What happens when the limit is reached? Or when no more memory can be allocated? 
Is it ok if one process sending to another problematic node will potentially exhaust all memory and crash the node?

If there was a configurable system_limit the sending process could for example get an exception from the send operation and crash with system_limit as reason.

I am just thinking here because I don't know how it is supposed to work. True asynchronous signaling will always be limited by the memory at some point.

Reply:
@KennethL Are you suggesting a command-line argument like `+zdbbl` but for a higher limit that may prevent an out of memory crash when the `async_dist` is used?  I was also concerned about this, but was happy the warning about use of the feature potentially causing a crash due to running out of memory (in absence of flow control or other limitations) was larger and more explicit in the documentation.